### ISSUE TITLE
ERR<eo>lib/eo/eo.c:611 in lib/edje/edje_object.eo.c:134: func 'edje_obj_signal_emit': obj_id=0x555f915eb270 is an invalid ref.

### ISSUE THREAD

**Issue by srakitnican** [JeffHoogland/moksha#75]
This messages appears when adjusting volume with a mouse scroll wheel.
```
May 13 09:01:10 machine /usr/libexec/gdm-x-session[6320]: ERR<eo>lib/eo/eo.c:611 in lib/edje/edje_object.eo.c:134: func 'edje_obj_signal_emit': obj_id=0x555f915eb270 is an invalid ref.
May 13 09:01:10 machine /usr/libexec/gdm-x-session[6320]: ERR<edje>lib/edje/edje_util.c:3968 file /usr/share/enlightenment/data/themes/default.edj, group e/modules/notification/main has a non-fixed part 'notification.swallow.app_icon'. Adding 'fixed: 1 1;' to source EDC may help. Continuing discarding faulty part.
```

**Comment by thewaiter** [comment]
Hmm, I am not able to reproduce this. I used the mouse wheel above the mixer gadget icon. Tell me more please.

**Comment by rbtylee** [comment]
OK I also am unable to duplicate this. But I am sure we should add the 'fixed:1 1;' part to the notification theme. It is present in the enlightenment part of the elementary theme and has been since 2015-06-16 with commit [notification: set fixed on app icon to eliminate edje calc error](https://git.enlightenment.org/core/elementary.git/commit/data/themes/edc/notification.edc?id=b4c4f339c3eb9196bb04601b3635a82fd0594f73). 

Hence I am backporting this: commit 13d064b.

Does this fix the issue?

**Comment by srakitnican** [comment]
The second message is gone by using 1bb5885c124c481cdb5eb249612ef59164aee816, the following message is still present though, but it seems it now appears only when notification pops up, it seems it is no longer updating when adjusting volume.
```
May 13 16:09:00 machine /usr/libexec/gdm-x-session[11254]: ERR<eo>lib/eo/eo.c:611 in lib/edje/edje_object.eo.c:134: func 'edje_obj_signal_emit': obj_id=0x55f917ce33c0 is an invalid ref.
```

**Comment by srakitnican** [comment]
I am also able to reproduce with command `notify-send "Test"`.

**Comment by rbtylee** [comment]
I am not getting the same error msgs exactly but I can duplicate the notification not always working when the volume is changed. Maybe the error msgs you are getting is a result of using gdm? Never thought about the possibility of different errors using different dms. But I will look for the commit that caused this issue and or a fix. May also play around with different dms...

**Comment by srakitnican** [comment]
You are right, the message doesn't appear when I start Moksha using `startx`. I can't reproduce notification not appearing every time though. To me it appears pretty consistently.
